### PR TITLE
Alert: Update max character length to 300

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
## Description

- Modified the message string to be a substring of 300 characters

References #

 #2242 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-alert/modus-alert.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export class ModusAlert {
           {this.type === 'warning' ? <IconWarning size={iconSize} /> : null}
         </div>
         <div class="message">
-          {this.message}
+          {this.message?.substring(0, 300)}
```
Comment (jmwood330t) [inline stencil-workspace/src/components/modus-alert/modus-alert.tsx:76]: Do we have unit tests that do boundary value analysis, i.e. tests with messages with lengths such as:

- Some number between (1, 298) inclusive
- 299
- 300
- 301
- Some number greater than 301

? And how should it handle the case when the length is greater than 300?

Comment (ElishaSamPeterPrabhu) [reply stencil-workspace/src/components/modus-alert/modus-alert.tsx:76]: When the text length is above 300 the characters are limited , so the test cases are written to check when the characters given are above 300 and the `div.message`'s text content length is 300 and the message is set to have a text with the text length to be 299 and is checked again to match it to 299.

**Review by cjwinsor — APPROVED** [review]
